index.js

```diff
@@ -256,6 +256,29 @@ const execa = (command, args, options) => {
 		return Promise.reject(error);
 	}
 
+	const originalKill = spawned.kill.bind(spawned);
+	spawned.kill = (signal, options = {}) => {
+		const killResult = originalKill(signal);
+		if (signal === undefined ||
+			(typeof signal === 'string' && signal.toUpperCase() === 'SIGTERM') ||
+			(typeof signal === 'number' && signal === os.constants.signals.SIGTERM)) {
+			const forceKill = !(options !== undefined && options.forceKill === false);
+
+			if (forceKill && killResult) {
```

Comment:
```suggestion
if (options.forceKill !== false && killResult) {
```

Then you can remove the `const forceKill = ...` line above.

Reply:
No, I still need it. Otherwise the behavior would be explicit. Now people need to specify `false` if they want to disable it. Your approach would require people to write `true` in the options.
Or we add default properties in the object but then we'd be screwed if the user decides to only specify the `forceKillAfter`.

Reply:
Yes that makes sense (basically the default value of `forceKill` is `true`) if I wrote `!options.forceKill` but it was `options.forceKill !== false`. When no `forceKill` option was passed, this will evaluate to false.

```js
const forceKill = !(options.forceKill === false);
if (forceKill && killResult) {
```

is the same as:

```js
if (!(options.forceKill === false) && killResult) {
```

which in turn is the same as:

```js
if (options.forceKill !== false && killResult) {
```